vochain/indexer/indexer.go

```diff
@@ -224,7 +216,7 @@ func (idx *Indexer) retrieveCounts() (map[uint8]uint64, error) {
 // still open (live) and updates all temporary data (current voting weight and live results
 // if unecrypted). This method might be called on a goroutine after initializing the Indexer.
 // TO-DO: refactor and use blockHeight for reusing existing live results
-func (idx *Indexer) AfterSyncBootstrap() {
+func (idx *Indexer) AfterSyncBootstrap(inTest bool) {
```

Comment:
Wouldn't it make more sense to have AfterSyncBootstrapTest() and AfterSyncBootstrap()
Then an `_innerAfterSyncBootStrap()` function with the actual code?

Reply:
I don't think that would really help - more lines of code and two more funcs, and we would never be calling the inner func directly either. I personally think one bool parameter and a conditional is pretty easy to follow.